docs/hermes.md

```diff
@@ -12,12 +12,10 @@ To learn more about what it is and how to use it, check out the [React Native](h
 Hermes is experimentally supported on Windows, generally providing better performance characteristics than the default Chakra engine.
 
 ### Enabling Hermes for new projects
-The easiest way to enable Hermes is to pass the `--useHermes` flag to `react-native-windows-init` when creating a new project. This will set up your project to use the Hermes engine and to generate bundles as Hermes bytecode instead of JavaScript.
+Pass the `--useHermes` flag to `react-native-windows-init` when creating a new project. This will set up your project to use the Hermes engine and to generate bundles as Hermes bytecode instead of JavaScript.
 
 ### Using Hermes in an existing project
-Hermes can be enabled in existing projects with a few edits.
-
-First, set the `UseHermes` property to `true` in the `ExperimentalFeatures.props` file in your project's `windows` directory:
+Set the `UseHermes` property to `true` in the `ExperimentalFeatures.props` file in your project's `windows` directory:
 

```

Comment:
should we say something like "if bringing your app from an older version of RNW, follow the instructions at [link to 0.64 instructions]"?